package.json

```diff
@@ -85,7 +85,7 @@
     "@mnajdova/enzyme-adapter-react-18": "^0.2.0",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.12",
-    "@mui/monorepo": "https://github.com/mui/material-ui.git#master",
+    "@mui/monorepo": "https://github.com/michaldudak/material-ui.git#esm-scripts",
```

Comment:
We should replace that one before merging I guess

Reply:
Of course, it's mentioned in the description. Just waiting for the original PR to be merged in material-ui repo. 😉